File: src/main/scala/com/github/mauricio/postgresql/messages/backend/AuthenticationResponseType.scala
package com.github.mauricio.postgresql.messages.backend

/**
 * User: mauricio
 * Date: 3/31/13
 * Time: 12:22 AM
 */
object AuthenticationResponseType extends Enumeration {
  type AuthenticationResponseType = Value
  val MD5, Cleartext, Ok = Value
}

File: postgresql-async/src/main/java/com/github/jasync/sql/db/postgresql/messages/backend/AuthenticationResponseType.kt
package com.github.jasync.sql.db.postgresql.messages.backend

//object AuthenticationResponseType : Enumeration {
//  type AuthenticationResponseType = Value
//  val MD5, Cleartext, Ok = Value
//}

enum class AuthenticationResponseType {
    MD5,
    Cleartext,
    Ok
}
